package.json

```diff
@@ -40,9 +40,9 @@
   },
   "gxDependencies": [
     {
-      "hash": "QmatUACvrFK3xYg1nd2iLAKfz7Yy5YB56tnzBYHpqiUuhn",
+      "hash": "QmPrKYPftocu7r4DhE3LorFgsJLGUkTPFLUqkS8SsuAXYn",
       "name": "go-ipfs",
-      "version": "0.4.14"
+      "version": "0.4.16-dev"
```

Comment:
in order to try this out, i forked from go-ipfs master, which is currently a whole minor version ahead of our version. hopefully we get some general improvements as well. (the hash above is our own gx published hash, which I have pinned to the relay node :)